.github/workflows/tests.yaml

```diff
@@ -302,7 +272,6 @@ jobs:
               --exclude libp2p-rpc-behaviour \
               --exclude node \
               --exclude mina-node-account \
-              --exclude vrf \
```

Comment:
Can you move this down to a comment at the bottom? Just helps us keep track of what's explicitly being run. Once we stop excluding most tests, we can delete the comment